### PR TITLE
added batch identifier field to create unique field

### PR DESCRIPTION
**Pull request by nidhi-sharma96**
created batch identifier field to save unique name in db. Kindly review

### CODE REVIEW

File: yota-api/src/main/java/com/yash/yotaapi/domain/TrainingMaster.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Entity
+@Table(name = "Training_Type_Master")
```
Comment (ramverma-github) [inline yota-api/src/main/java/com/yash/yotaapi/domain/TrainingMaster.java:21]: Please remove this changes, remove entire line.

File: yota-api/src/main/java/com/yash/yotaapi/serviceimpl/BatchServiceImpl.java [hunk 2]
```diff
@@ -26,28 +27,23 @@ public class BatchServiceImpl implements BatchService {
...
 	public Batch createBatch(Batch batch) {
-
-
 		batch.setBatchName(batch.getBatchName().toUpperCase());
-		// batch.setBatchIdentifier(batch.getBatchIdentifier().toUpperCase());
+		String input = batch.getBatchName().toUpperCase();
+		String[] parts = input.split("-");
+		if (parts.length > 1) {
+			String wordAfterHyphen = parts[1];
+			batch.setBatchIdentifier(UniqueNameGenerator.generateUniqueName("Pankaj Sharma", wordAfterHyphen));
```
Comment (ramverma-github) [inline yota-api/src/main/java/com/yash/yotaapi/serviceimpl/BatchServiceImpl.java:39]: Please update to take this hardcode from UI

**Review by ramverma-github — COMMENTED** [review]
Please resolve comments and do the changes.

File: yota-api/src/main/java/com/yash/yotaapi/domain/TrainingMaster.java (hunk 1 above)
Comment (nidhi-sharma96) [reply yota-api/src/main/java/com/yash/yotaapi/domain/TrainingMaster.java]: resolved

File: yota-api/src/main/java/com/yash/yotaapi/serviceimpl/BatchServiceImpl.java (hunk 2 above)
Comment (nidhi-sharma96) [reply yota-api/src/main/java/com/yash/yotaapi/serviceimpl/BatchServiceImpl.java]: resolved

**Review by ramverma-github — APPROVED** [review]
Looks good.

**Pull request merged**